src/storage/addresses/addressesSlice.ts

```diff
@@ -140,9 +147,7 @@ const addressesSlice = createSlice({
           return {
             id: hash,
             changes: {
-              ...details,
-              tokens,
-              transactions: uniq([...address.transactions, ...transactionHashes]),
+              transactions: uniq(address.transactions.concat(transactions.map((tx) => tx.hash))),
```

Comment:
Nit: we usually use spreading to do this, which is easier to read IMO. 

```ts
uniq([...address.transactions, ...map(transactions, 'hash')])
``` 

or 
```ts
uniq([...address.transactions, ...transactions.map((tx) => tx.hash)])
``` 

(because I'm sad about the typing of lodash's property shorthand `map` which isn't strict...)

Reply:
Fixed!